builder-run

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 #
-# A utility script for executing arbitrary local scripts via docker.
+# A utility script for executing arbitrary local scripts via podman.
```

Comment:
This script is from coreos/tectonic-installer#609, but doesn't seem to have ever been used anywhere.  With `installer/scripts` gone since coreos/tectonic-installer#3067, I'd guess we can just drop `builder-run` entirely.

Reply:
:+1: to drop whatever we can :)

Reply:
> ... I'd guess we can just drop builder-run entirely.

I've spun this off into #229.